luxtronik/datatypes.py

```diff
@@ -86,6 +86,12 @@ class Seconds(Base):
     measurement_type = "seconds"
 
 
+class Minutes(Base):
+    """MinutesSeconds datatype, converts from and to Minutes."""
```

Comment:
`Seconds` is probably a leftover in this string, should be removed:

```suggestion
    """Minutes datatype, converts from and to Minutes."""
```